drivers/wifi/nrf_wifi/Kconfig.nrfwifi

```diff
@@ -828,4 +828,16 @@ config NRF_WIFI_COEX_DISABLE_PRIORITY_WINDOW_FOR_SCAN
 	help
 	  Enable this configuration to disable priority window for scan
 	  in the case of coexistence with Short Range radio.
+
+config NRF_WIFI_ZERO_COPY_TX
+	bool "Zero copy TX"
+	select NET_L2_ETHERNET_RESERVE_HEADER
+	default y
+	help
+	  Enable this configuration to use zero copy Transmit path.
+	  The driver will use the network buffer directly for transmission
+	  without copying the data to the driver's buffer. This reduces the
+	  driver heap memory usage without much impact on the performance.
+	  This is still an experimental feature.
```

Comment:
Perhaps it should `select EXPERIMENTAL` have `[EXPERIMENTAL]` suffix in the description then?

Reply:
Yeah, I had intentionally left it out as it's enabled y default and could affect SML, but I guess SML (NCS) is now manual, so, will add that label.